src/main/java/Contacts0.java

```diff
@@ -146,4 +144,12 @@ public static void main(String[] args) {
         }
     }
 
+    private static void showWelcomeScreen() {
```

Comment:
You can add a documentation here :3

Reply:
Ya i agree on this one too! A documentation would be good!